timescale_vector/client.py

```diff
@@ -433,13 +434,29 @@ def build_query(self, params: List) -> Tuple[str, List]:
                     params.append(value)
                     continue
 
+                # if array containment
+                if operator == "@>" and isinstance(value, list):
+                    field_cast = ''
+                    if len(value) == 0:
```

Comment:
what does this even mean? shouldn't an empty list be an error?

Reply:
Oddly enough, an empty array seems to match everything. Yeah, that's probably not desirable.
<img width="763" alt="image" src="https://github.com/timescale/python-vector/assets/4974668/1c05d856-5b07-4134-a457-4c54b3b76aaf">

Reply:
latest commit throws ValueError on empty list/tuple